llvm/include/llvm/IR/DataLayout.h

```diff
@@ -299,6 +302,17 @@ class DataLayout {
     return ManglingMode == MM_WinCOFFX86;
   }
 
+  uint64_t getNonZeroValueForAddrSpace(uint64_t AddrSpace) {
+  auto It = AddrSpaceToNonZeroValueMap.find(AddrSpace);
+  if (It == AddrSpaceToNonZeroValueMap.end())
+    return 0x000000000;
+  return AddrSpaceToNonZeroValueMap[AddrSpace];
+  }
+
+  void setNonZeroValueForAddrSpace(uint64_t AddrSpace, uint64_t Value) {
```

Comment:
ditto, use something like `setNullPointerValue`

Reply:
renamed to setNullPointerValue.